stable-baselines3/sb3.py

```diff
@@ -120,6 +121,18 @@ def parse_args(input=sys.argv[1:]):
             default=-1,
             type=int,
         )
+        parser.add_argument(
```

Comment:
Don't add arg twice. Since it is mutual to both inference and train add to parser right after its initilializaiton on line ~102

Reply:
Good call, the things we learn when working together.